pkg/infrastructure/server/cluster.go

```diff
@@ -1,108 +0,0 @@
-package server
```

Comment:
Farewell sweet JSON API, but you were a burden to maintain along the gRPC API, hope to see you again soon